rules/configuration.smk

```diff
@@ -75,11 +91,6 @@ samples.index = samples.index.map(str)
 if config.get('peak_caller', False):
     config['peak_caller'] = {k: v for k,v in config['peak_caller'].items()}
 
-    # if hmmratac peak caller, check if all samples are paired-end
-    if 'hmmratac' in config['peak_caller']:
-        assert all([config['layout'][sample] == 'PAIRED' for sample in samples.index]), \
-        "HMMRATAC requires all samples to be paired end"
-
```

Comment:
Either we fully drop hmmratac, or we keep this. Not a bit of both

Reply:
Turns out this section is moved, not deleted :)

Reply:
Ooops :angel: